Updates to Installing CF on OCP guide

dayleparker opened the pull request:
Hi Chris,
I'm still waiting on confirmation on a few questions from SMEs, but was thinking it would be good to have your peer review sooner if possible, to ensure we get this update out for GA.

This comment probably provides the most context of what's changed since 4.5, a lot of which I found in the 4.6 enablement presentation: https://bugzilla.redhat.com/show_bug.cgi?id=1536933#c6

Would you mind reviewing for structure/grammar/etc, and let me know if you spot anything needing a fix. A preview is linked in Comment #6 and 7 of the bug.

Many thanks,
Dayle

cbudz commented on doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc:
```diff
@@ -0,0 +1,169 @@
+$ oc create -f cfme-restore-job.yaml
+----
+
+After [INSERT RESULT], you can scale pods back up.
```
Is [INSERT RESULTS] intentional? if this is a placeholder of I'm misunderstanding for lack of my own personal knowledge please ignore this comment.

cbudz commented on doc-Installing_on_OpenShift_Container_Platform/topics/post-installation.adoc:
```diff
@@ -0,0 +1,169 @@
+[[post-installation]]
+== Managing {product-title_short} with OpenShift
+
+This section includes common tasks to manage your {product-title_short} deployment from OpenShift.
```
Should we have the full product name here since its the first instance in the section?

dayleparker commented:
@cbudz Hey Chris,

Thanks so much for the review and comments -- I've fixed those two things, writing out the full product nameat the beginning of the post-install section, and have edited out the [INSERT TEXT] part as it was just a placeholder for info (exactly as you thought) I was hoping to get before GA.

I made a few more changes, like removing any other placeholder text, and the SCCs table, as I've found this info can be pretty easily looked up with some OCP commands. I've added a few other 'further reading' links, as I found some instructions in the OCP 3.7 release notes for doing the auto-install of CloudForms... huzzah! 

If it looks OK, please feel free to merge. I'm going to reach out to a few more people who might be able to answer my questions about where to get the images and the .yaml files in the meantime. Or if you spot anything else I've missed, please let me know.

Cheers,
Dayle

cbudz commented:
Hi @dayleparker,

Changes look good. merging this PR.

-Chris
